content/docs/setup/kubernetes/install/helm/index.md

```diff
@@ -10,11 +10,27 @@ aliases:
 icon: helm
 ---
 
-Follow this path to install and configure an Istio mesh using Helm.
+Follow this path to install and configure an Istio mesh for in-depth evaluation or production use.
 
-**This path is recommended for production environments.** This path offers rich
-customization of the Istio control plane and of the sidecars for the Istio data
-plane.
+These instructions use [Helm](https://github.com/helm/helm) charts that provide rich
+customization of the Istio control plane and of the sidecars for the Istio data plane.
+You can start with one of Istio's built-in configuration profiles and then further customize
+the configuration for your specific needs.
+The following built-in configuration profiles are currently available:
+
+1. **minimal**: the minimal set of components necessary to use Istio's [traffic management](/docs/tasks/traffic-management/) features.
+
+1. **demo**: configuration suitable to run the [Bookinfo](/docs/examples/bookinfo/) application and associated tasks.
+    This is the same configuration that is installed with the [Quick Start](/docs/setup/kubernetes/install/kubernetes/) instructions, only using helm has the advantage
+    that you can more easily enable additional features if you wish to explore more advanced tasks.
+
+1. **default**: enables components according to the default [Installation Options](/docs/reference/config/installation-options/)
+    (recommend for production deployments).
+
```

Comment:
should we add the remote profile?  e.g. values-istio-remote.yaml

Reply:
and sds enabled profile?  values-istio-sds-auth.yaml

Reply:
I think so, yes, but in a future PR (probably after 1.1) when we have a good inventory of what all the profiles are, like the table requested in #3460